packages/react-core/src/components/LabelGroup/examples/LabelGroupEditableAdd.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { LabelGroup, Label } from '@patternfly/react-core';
+
+export const LabelGroupEditableAdd: React.FunctionComponent = () => {
+  const [idIndex, setIdIndex] = React.useState<number>(3);
+  const [labels, setLabels] = React.useState<any>([
+    { name: 'Label 1', id: 0 },
+    { name: 'Label 2', id: 1 },
+    {
+      name: 'Label 3',
+      props: {
+        isEditable: true,
+        editableProps: {
+          'aria-label': 'label editable text'
+        }
+      },
+      id: 2
+    }
+  ]);
+
+  const onClose = (labelId: string) => {
+    setLabels(labels.filter((l: any) => l.id !== labelId));
+  };
+
+  const onEdit = (nextText: string, index: number) => {
+    const copy = [...labels];
+    copy[index] = { name: nextText, props: labels[index].props };
+    setLabels(copy);
+  };
```

Comment:
For the bug shown in the video above, it looks like the issue occurs here; the `label.id` ends up getting removed after editing.

Reply:
Ah good catch, I missed preserving that when I updated everything to use an id instead of name.